.github/workflows/image-build.yml

```diff
@@ -0,0 +1,32 @@
+name: Build Image Test
+
+on:
+  pull_request:
+    branches: [main, release/**]
+
+env:
+  OWNER: hashgraph
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Qemu
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          driver-opts: network=host
+
+      - name: Build image
+        uses: docker/build-push-action@v2
+        with:
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          platforms: linux/amd64, linux/arm64
+          push: false
```

Comment:
Q: Is there anything we can do to test the image or no?

Reply:
Yes, just added steps for that...

![image](https://github.com/hashgraph/hedera-json-rpc-relay/assets/123040664/3773e504-1f3a-4e53-92c2-4612134e73fa)